src/gui_state.cpp

```diff
@@ -1011,8 +1012,8 @@ void gui_state::gui() {
   update();
   gui_main(*this);
   gui_locations(*this);
-  gui_cell(*this);
   gui_traces(*this);
+  gui_cell(*this);
```

Comment:
Why the shuffle? 

The open/shown panes are controlled by the ini file stored under default.ini. If you want to change these setting copy said ini file to default.ini and commit it.